user/billing-overview.md

```diff
@@ -265,7 +263,7 @@ Each new user that subscribes to the Free Trial Plan is automatically granted 10
 
 
 
-### Usage - User Licenses
+### Usage - User Licenses (Standard and Pro Plan)
```

Comment:
@stan-travis actually Pro Plan is not in use anywhere else than a manually set custom contract. I would suggest to change `(Standard and Pro Plan)` to `Usage Plan w/o susbscription` to keep naming consistent with the FAQ